tests/conftest.py

```diff
@@ -689,9 +694,59 @@ def map_data_line(line):
     _write_log_line(f"After run test case {request.function.__name__}:")
 
 
+@pytest.fixture
+async def enable_auth() -> bool:
+    """
+    A fixture that indicates whether the server_config fixture should
+    set server.auth to true or false.
+    """
+    return False
+
+
+@pytest.fixture
+async def access_policy() -> str:
+    """
+    A fixture that returns the access policy configured by the server_config fixture.
+    """
+    return """
+        package policy
+
+        # Allow everything
+        default allowed:=true
+    """
+
+
+@pytest.fixture(scope="session")
+async def path_policy_engine_executable() -> str:
+    """
+    Returns the path to the Open Policy Agent executable.
+    This method caches the executables to prevent slow setup times of the test suite.
+    """
+    cache_dir = os.path.abspath(os.path.join(__file__, "..", "data", "opa_executables.cache"))
+    os.makedirs(cache_dir, exist_ok=True)
+    cache_file = os.path.join(cache_dir, "opa-1.3.0")
+    if not os.path.exists(cache_file):
+        with open(cache_file, "wb") as fp:
+            req = requests.get("http://file.ii.inmanta.com/software/opa/opa-1.3.0", stream=True)
```

Comment:
Do we have a plan to manage this version?

Reply:
I currently put the version of opa in the `inmanta/__init__.py`. But I don't think there is an automated way to update the version numbers. Unless we develop something ourselves.